gc/gc-base/src/main/java/org/projectnessie/gc/base/SerializableFunction1.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.gc.base;
+
+import java.io.Serializable;
+
+/** Interface that makes scala.Function1 as Serializable. */
+public interface SerializableFunction1<T, U> extends scala.Function1<T, U>, Serializable {}
```

Comment:
can you try using static methods instead of passing this function?

Reply:
tried, replied above to similar coment.